paragraph/scheduling/graph_scheduler.h

```diff
@@ -88,6 +89,10 @@ class GraphScheduler {
   // API is used.
   double current_time_;
 
+  // Logger that logs to file all the instruction timings during the graph
```

Comment:
Leave the file part of the implementation in the Logger class. Just state that all information about the instruction timing is passed to the Logger.

Reply:
fixed